src/platform/i386/boot_comp.c

```diff
@@ -15,14 +15,17 @@ extern u8_t *boot_comp_pgd;
 int boot_nptes(unsigned int sz) { return round_up_to_pow2(sz, PGD_RANGE)/PGD_RANGE; }
 
 int
-boot_pgtbl_mappings_add(struct captbl *ct, pgtbl_t pgtbl, capid_t ptecap, const char *label,
+boot_pgtbl_mappings_add(struct captbl *ct, capid_t pgdcap, capid_t ptecap, const char *label,
```

Comment:
I don't understand what this means.  I see no physical address.  I only see a type that was made ambiguous.  I don't understand your statement as it seems trivially wrong.